src/Images/Pexels.php

```diff
@@ -26,13 +27,73 @@ class Pexels {
 	 * @return array|\WP_Error Array of images, or WP_Error if the request failed.
 	 */
 	public function get_images( $ai_connection, $token, $business_description ) {
-		$search_term = $this->define_search_term( $ai_connection, $token, $business_description );
+		if ( str_word_count( $business_description ) === 1 ) {
+			$search_term = $business_description;
```

Comment:
It is not a strong opinion, but I would remove this check, given that it is a very rare use case, I would keep the logic simple.